test/test-server/NextApi.TestClient/TestService.cs

```diff
@@ -24,7 +24,7 @@ public interface ITestService : INextApiService
         Task AsyncVoidDenied();
         string GetCurrentUser();
         Task<string> UploadFile(Stream fileStream, string fileName);
-        Task<NextApiFileResponse> GetFile(string path);
+        Task<NextApiFileResponse> GetFile(string path, bool throwException = false);
```

Comment:
I think, that it is better to add another mock method, instead of modifying existing one, that will have return type of `Task<NextApiFileResponse>`, will accept only path, and will always throw an exception, if you really need it.